pywr/recorders/_recorders.pyx

```diff
@@ -1972,6 +1984,18 @@ cdef class AnnualCountIndexThresholdRecorder(Recorder):
 
         if self.exclude_months is not None and ts.month in self.exclude_months:
             return
+        
+        # include a range of dates within a year
+        if self.include_from_month is None or self.include_from_day is None:
+            include_from = 1
```

Comment:
This should be cdef'd at the top of the method. 

```
cdef int include_from
cdef int include_to
```

Reply:
Added